src/api/app/services/notification_service/notifier.rb

```diff
@@ -59,5 +62,13 @@ def notifiable_exists?
       notifiable_id = @event.parameters_for_notification[:notifiable_id]
       notifiable_type.exists?(notifiable_id)
     end
+
+    def create_report_notification?(event:, subscriber:)
```

Comment:
Not sure about this one. Why do we need all those checks?

Reply:
I wanted to avoid that users keep receiving emails after they left beta... I thought it doesn't hurt